crates/types/src/config/common.rs

```diff
@@ -628,6 +628,7 @@ enum MetadataClientKindShadow {
     #[serde(alias = "embedded")]
     Native {
         address: Option<AdvertisedAddress>,
+        #[serde(default)]
```

Comment:
why does this need default?

Reply:
It failed to deserialize without it if `addresses` was not provided. It's so weird since this used to work when I first added it.